From 808c07843f4591fc71b19a3e2ffeca2383d396e4 Mon Sep 17 00:00:00 2001
From: Johannes Lorenz <j.git@lorenz-ho.me>
Date: Mon, 18 Nov 2019 19:10:42 +0100
Subject: [PATCH 1454/1936] Re-enable port-checker

Port checker did not work because a too old rtosc submodule has been merged
into zyn. The recent commit fixed the issues by updating rtosc.
---
 src/Tests/CMakeLists.txt | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/Tests/CMakeLists.txt b/src/Tests/CMakeLists.txt
index 0dfcf6b5..98dd4e47 100644
--- a/src/Tests/CMakeLists.txt
+++ b/src/Tests/CMakeLists.txt
@@ -72,10 +72,10 @@ target_link_libraries(EffectTest ${test_lib})
 add_executable(ins-test InstrumentStats.cpp)
 target_link_libraries(ins-test ${test_lib} rt)
 
-#if(LIBLO_FOUND)
-#    cp_script(check-ports.rb)
-#    add_test(PortChecker check-ports.rb)
-#endif()
+if(LIBLO_FOUND)
+    cp_script(check-ports.rb)
+    add_test(PortChecker check-ports.rb)
+endif()
 
 add_executable(save-osc SaveOSC.cpp)
 target_link_libraries(save-osc
-- 
2.47.0

